### ISSUE TITLE
Prefix tag search doesn't work

### ISSUE THREAD

**Issue by garberg** [SpriteLink/NIPAP#1008]
The prefix tag search doesn't work any more:
```
$ nipap address list "#foo"
Searching for prefixes in any VRF...
Command failed: <type 'exceptions.TypeError'>:string indices must be integers, not str
```
Searching for tags in pools and VRFs still works though.

**Comment by gitkevhub** [comment]
so should one be able to search for tags in prefix?

**Comment by garberg** [comment]
Absolutely, this bug was fixed in #1012 after which this issue was closed. If you have problems with searching for tags, please open a new issue.
